src/symbolize/gimli/mmap_unix.rs

```diff
@@ -4,14 +4,19 @@ use core::ops::Deref;
 use core::ptr;
 use core::slice;
 
+#[cfg(not(target_os = "linux"))]
+use libc::mmap as mmap64;
+#[cfg(target_os = "linux")]
```

Comment:
should only use mmap64 on glibc, see <https://github.com/rust-lang/rust/issues/94173#issuecomment-1358619183>